### PR TITLE
worker: add log for delegate register and destroy.

### PR DESCRIPTION
**Pull request by siddontang**
PTAL @BusyJay @hhkbp2 @javaforfun

### CODE REVIEW

File: src/raftstore/store/worker/apply.rs [hunk 1]
```diff
@@ -1293,6 +1293,9 @@ impl Runner {
         let region_id = s.region.get_id();
         let term = s.term;
         let delegate = ApplyDelegate::from_registration(self.db.clone(), s);
+        info!("{} register to apply delegates at term {}",
+              delegate.tag,
+              delegate.term);
```
Comment (hhkbp2) [inline src/raftstore/store/worker/apply.rs]: Is it necessary to log the region info?

**Comment by hhkbp2** [comment]
Rest LGTM

**Comment by BusyJay** [comment]
LGTM

File: src/raftstore/store/worker/apply.rs (hunk 1 above)
Comment (siddontang) [inline src/raftstore/store/worker/apply.rs]: Seem not necessary here. Outer always logs the region info before registration. 

@hhkbp2

**Comment by hhkbp2** [comment]
LGTM

**Pull request merged**